ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -51,6 +51,21 @@
     <input type="hidden" name=":formstart" value="${resource.path}"/>
     <input type="hidden" name="_charset_" value="utf-8"/>
     <input data-sly-test="${container.redirectUrl}" type="hidden" name=":redirect" value="${container.redirectUrl}"/>
+    <div data-sly-test="${container.properties['fd:hamburgerMenu']}" class="cmp-adaptiveform-container-hamburger-menu-container">
```

Comment:
Why do you need a getter for hamburger menu, when you could access this using properties. We could remove the getter from the interface completely if not required

Reply:
It is required as per the code standard and reusability.